### PR TITLE
UF-7792: Added port config to service

### PR DESCRIPTION
**Pull request by josc04**
- Added port config to service
- Added validation on future dates to expire and reminder dates

### CODE REVIEW

File: src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java [hunk 1]
```diff
@@ -30,8 +33,10 @@ public class SigningRequest {
 	@Schema(description = "Optional descriptive name for the document that is to be signed.", example = "Employment contract")
 	private String name;
 
-	@Schema(description = "The date and time when the signing request expires. Format is yyyy-MM-dd'T'HH:mm:ss.SSSXXX", example = "2024-01-31T12:30:00.000", requiredMode = REQUIRED)
+	@Schema(description = "The date and time when the signing request expires.", example = "2021-12-31T23:59:59Z", requiredMode = REQUIRED)
```
Comment (cheezi0747) [inline src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java:36]: If this is not set Comfact will set a default of 30days. Do we need this to be required?

Comment (josc04) [reply src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java:36]: Ok, didn't know that. I'll check with Jakob if expiredate should or should not be mandatory.

Comment (josc04) [reply src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java]: I've spoken to Jakob who has decided that expire attribute should be optional. Adjustments has been made.

**Review by cheezi0747 — APPROVED** [review]

File: src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java (hunk 1 above)
Comment (foobar-1337) [reply src/main/java/se/sundsvall/esigning/api/model/SigningRequest.java]: image

**Pull request merged**